app/models/comment.rb

```diff
@@ -0,0 +1,16 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: 'AuthorId'
```

Comment:
- [ ] Great job adding the associations, but you are missing something here. There's an Author model existing which is an alias to the User model. Therefore your association should be something like 
`belongs_to :author, class_name: 'User'`